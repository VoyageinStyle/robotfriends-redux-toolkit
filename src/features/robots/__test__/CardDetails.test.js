import React from "react";
import renderer from "react-test-renderer";

import CardDetail from "../CardDetails";

it("shows extra details of Card component", () => {
  const tree = renderer.create(
    <CardDetail
      robot={{
        id: 1,
        company: {
          name: "zzz",
        },
        address: {
          city: "bla",
        },
      }}
    />
  );
  expect(tree).toMatchSnapshot();
});
