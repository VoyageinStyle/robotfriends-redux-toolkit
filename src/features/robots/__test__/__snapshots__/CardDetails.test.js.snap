// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows extra details of Card component 1`] = `
<div
  className="hidden"
>
  <br />
  <p>
    <b>
      Company:
    </b>
     
    zzz
  </p>
  <p>
    <b>
      Location:
    </b>
     
    bla
     
  </p>
  <p>
    <b>
      Catch phrase: 
    </b>
  </p>
  <em />
</div>
`;
